.tooling/markdownlint/custom-formatting.js

```diff
@@ -0,0 +1,56 @@
+import { forEachLine, getLineMetadata } from "markdownlint-rule-helpers";
+
+const customFormattingRules = [
+  {
```

Comment:
@mattleff The Lao forbidden terms should apply to the Lao md files and the Thai forbidden terms should apply to the Thai md files. They should always apply so that spelling errors are caught early in the translation process. 

There are probably some character combinations that should always be forbidden. Like space before closing parentheses. This would apply everywhere. 

Does this answer your question or do I not understand the question?